geemusic/intents/selection.py

```diff
@@ -158,3 +158,26 @@ def play_artist_radio(artist_name):
 
     speech_text = "Playing music from your personalized station"
     return audio(speech_text).play(stream_url)
+
+@ask.intent("GeeMusicQueueSongIntent")
+def queue_song(song_name, artist_name):
+    api = GMusicWrapper.generate_api()
+
+    app.logger.debug("Queuing song %s by %s" % (song_name, artist_name))
+
+    if len(queue.song_ids) == 0:
+        return statement("You must first play a song")
+
+    # Fetch the song
+    song = api.get_song(song_name, artist_name=artist_name)
+
+    if song is False:
+        return statement("Sorry, I couldn't find that song")
+
+    # Queue the track in the list of song_ids
+    queue.song_ids.append(song['storeId'])
```

Comment:
I don't know that this is the best way to do this, as it might cause `queue.current_track()` (see [here](https://github.com/stevenleeg/geemusic/blob/master/geemusic/utils/music_queue.py#L35-L39)) to break, since the track definition won't be in the queue. This would prevent the "what's currently playing?" intent from functioning (and might cause the server to crash).

We might want to add something like `queue.enqueue_track(song)` which handles storing the track definition in addition to adding the ID to the queue.